galgebra/mv.py

```diff
@@ -170,6 +170,14 @@ def characterise_Mv(self):
     # positionally. When python 3.8 is the lowest supported version, we can
     # switch to using the / syntax from PEP570
 
+    @staticmethod
+    def _make_blade(ga, __name, __grade, **kwargs):
+        if isinstance(__name, str) and isinstance(__grade, int):
+            return reduce(Mv.__xor__, [ga.mv('%s%d' % (__name, i), 'vector') for i in range(__grade)],
+                          ga.mv(1, 'scalar')).obj
+        else:
+            raise TypeError("name must be a string and grade must be an int")
```

Comment:
This is a nice idea, but unfortunately it won't work with `ga.make_grad(some_blade)`, as the coefficients are no longer single variables.

Perhaps it's worth cherry-picking into master anyway.